.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Add "/node_modules" to .gitignore, sometimes this directory can be added by accident while setting up linters. Just in case ;)